tensorflow/compiler/xla/client/computation_builder.h

```diff
@@ -211,9 +211,11 @@ class ComputationBuilder {
   //
   // Note that "limit" means up-to-but-not-including; i.e. [start, limit) in 1D
   // range notation.
+  // The stride parameter determines the stride over the slice
```

Comment:
I talked about it with some folks and the consensus seemed to be to prefer positive strides only for now. The reasoning is:
a) we want to keep XLA minimal where possible, since it makes life easier for backend authors --- fewer cases to implement,
b) it doesn't add any expressive power, since you can use Rev() to reverse dimensions, and it doesn't add any efficiency because we can fuse Rev() and Slice().
c) it's easier to add features later rather than to remove them, so we can always add support for negative strides if we think of a good reason why.